roles/check-instance/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+allow_install_on_vm: false
+
+ansible_user: ansible
+
+required_cpu_cores: 4
```

Comment:
Why 4?  I've successfully run oracle on a single core without too much trouble

Reply:
This came from the Oracle DBA team. I can remove or anyone can change to 1 but for BM there are no cases where this will fail.

Reply:
I changed it to 1 so any combination is valid. Still the check is there, someone might change it to something else and use it.